invenio_rdm_migrator/streams/runner.py

```diff
@@ -96,10 +103,20 @@ def run(self):
                 stream.run()
                 # sucessfully finished stream run, now we can dump that stream cache
                 for name, cache in self.cache.items():
-                    if name == "communities":  # FIXME: implement communities cache
+                    if (
+                        name == "communities" or name == "max_pid"
+                    ):  # FIXME: implement communities cache
                         continue
                     cache_file = self.cache_dir / f"{name}.json"
                     cache.dump(cache_file)
             except Exception:
                 self.logger.error(f"Stream {stream.name} failed.", exc_info=1)
                 continue
+        # dump the max pid generated in pids cache
+        pid_cache = self.cache.get("max_pid")
+        # call global pid_pk to update the cache with the max value
+        max_pid_value = pid_pk()
+        print("Max pid value", max_pid_value)
+        pid_cache.update("max_value", max_pid_value)
+        cache_file = self.cache_dir / "max_pid.json"
+        pid_cache.dump(cache_file)
```

Comment:
we need this dump so it can be manually changed in `pid_pk` generation for 2nd run of the draft stream